### ISSUE TITLE
[Feature request] SSL cert table for posix

### ISSUE THREAD

**Issue by friedbutter** [facebook/osquery#3811]
We would like to have a table called ssl_cert. The table should have the following schema:

```
issued_common_name
issued_organization
issued_organization_unit
issued_serial_number
issuer_cn
issuer_organization 
issuer_organization_unit
valid_from 
valid_to 
sha256_fingerprint
sha1_fingerprint
```
How we want to use it:

```select * from ssl_cert where domain = 'www.facebook.com';```

It will **fetch the certificate from the domain** and parse out the cert. 

You should use openssl lib since it already in osquery. 

For an initial draft, make this table for posix (macos and linux) systems.

**Comment by micheal-o** [comment]
I will work on this.

**Comment by muffins** [comment]
This was addressed in #3839
